src/segment-loader.js

```diff
@@ -754,6 +755,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     if (request === this.xhr_.segmentXhr) {
       // the segment request is no longer outstanding
       this.xhr_.segmentXhr = null;
+      segmentInfo.startOfAppend = Date.now();
```

Comment:
Unfortunately bandwidth is only the segment request. So the rest of the process is something I current consider "overhead" so I can capture key response overhead.